substra/cli/interface.py

```diff
@@ -445,6 +445,7 @@ def add_algo(ctx, data, output_format, config, profile, user, verbose):
 @click.option('--data-samples-path', 'data_samples', required=True,
               type=click.Path(exists=True, resolve_path=True, dir_okay=False),
               callback=load_json_from_path)
+@click.option('--in-models-keys')
```

Comment:
Could add more details about this option? The type for example, whether it's multiple or not etc.